apps/server/src/modules/class/service/class.service.ts

```diff
@@ -13,6 +13,13 @@ export class ClassService {
 		return classes;
 	}
 
+	public async findAllByUserId(userId: EntityId): Promise<Class[]> {
+		const classes: Class[] = await this.classesRepo.findAllByUserId(userId);
+
+		return classes;
+	}
+
+	// FIXME There is no usage of this method
```

Comment:
When? In a new ticket?

Reply:
How should I know. This is not my method. I think it was written by passcon for the deletion of users, but it is not used atm.

Reply:
Alright